ansible/roles/node/templates/kubelet.j2

```diff
@@ -11,7 +11,7 @@ KUBELET_ADDRESS="--address=0.0.0.0"
 KUBELET_HOSTNAME="--hostname-override={{ inventory_hostname }}"
 
 # location of the api-server
-KUBELET_API_SERVER="--api-servers=https://{{ groups['masters'][0] }}:{{ kube_master_api_port }}"
+KUBELET_API_SERVER="--api-servers=https://{{ kube_apiserver_endpoint }}:{{ kube_master_api_port }}"
```

Comment:
I think this can change to the following so we do not need to introduce the kube_apiserver_endpoint var:

```
KUBELET_API_SERVER="--api-servers=https://{% if groups['masters']|length > 1 %}{{ kube_apiserver_vip }}{% else %}{{ groups['masters'][0] }}{% endif %}:{{ kube_master_api_port }}"
```